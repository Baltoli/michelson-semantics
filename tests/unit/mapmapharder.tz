$Instructions { MAP { CDR ; ADD ; DUP } } ; 
$Stack ListItem(#Map(1 |-> 3 5 |-> 2 12 |-> 20, int, int)) ListItem(0);
