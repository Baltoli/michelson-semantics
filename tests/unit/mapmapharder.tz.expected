<generatedTop>
  <k>
    .
  </k>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <stack>
    Stack ( ListItem ( 1 |-> 3
    5 |-> 5
    12 |-> 25 )
    ListItem ( 0 ) )
  </stack>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( -1 )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( -2 )
  </sourceaddr>
  <senderaddr>
    #Address ( -3 )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
